src/_tags

```diff
@@ -3,5 +3,5 @@ true: thread
 true: package(oUnit)
 true: package(lwt)
 true: package(lwt.unix)
-true: package(logs.lwt)
+true: package(lwt.log)
```

Comment:
This does not compile for me:
ocamlfind: Package `lwt.log' not found

(while the original compiles)
```
 opam list
# Installed packages for 4.06.1:
base-bigarray        base  Bigarray library distributed with the OCaml compiler
base-bytes           base  Bytes library distributed with the OCaml compiler
base-threads         base  Threads library distributed with the OCaml compiler
base-unix            base  Unix library distributed with the OCaml compiler
conf-m4                 1  Virtual package relying on m4
cppo                1.6.4  Equivalent of the C preprocessor for OCaml programs
jbuilder       1.0+beta20  Fast, portable and opinionated build system
logs                0.6.2  Logging infrastructure for OCaml
lwt                 4.0.1  Promises, concurrency, and parallelized I/O
lwt_log             1.1.0  Lwt logging library (deprecated)
ocamlbuild         0.12.0  OCamlbuild is a build system with builtin rules to easily build most OCaml projects.
ocamlfind           1.8.0  A library manager for OCaml
ounit               2.0.8  Unit testing framework loosely based on HUnit. It is similar to JUnit, and other XUnit testing frameworks
result                1.3  Compatibility Result module
topkg               0.9.1  The transitory OCaml software packager
```

Reply:
Versions of `lwt` before 4.0.0 published a sublibrary, `log`, which is no longer available in `lwt` versions >= 4.0.0 .  That code is now in `lwt_log` (although it is also deprecated, and [the lwt changelog](https://github.com/ocsigen/lwt/blob/master/CHANGES#L21) recommends using the `logs` library instead, which publishes a module `Logs_lwt`).